docs/pages/accessibility.mdx

```diff
@@ -0,0 +1,104 @@
+
+## Making UI accessible
+
+FluentUI React Native components are accessible by default meaning you only need to provide minimal customizations to make
+your UI accessible. Here you will find information the folowing components: Properties, Roles, States, and Actions.
+
+### Properties
+
+#### accessible
+When ```true```, indicates that the view is an accessible element meaning it receives accessibility focus. By default, all 
+controls are accessible. 
+
+```tsx
+<Text accessible={true}>first</Text>
+<Text accessible={false}>second</Text>
+```
+
+In the above example, 'first' recieves accessiblity focus but 'second' does not. 
+
+
+#### accessibilityLabel
+When something is accessible, it's good practice to set an ```accessiblityLabel``` so the user knows what element they have
+selected. Narrator will read out this string when the user places focus on the control.
+
+To use, set the ```accessibilityLabel``` property to a custom string:
+
+```tsx
+<Button content="Click me" accessibilityLabel="Press me" />
+```
+
+In the above example, Narrator will read "Press me" instead of "Click me".
+
+#### accessibilityHint
+
+An accessibility hint helps users understand what will happen when they preform an action on the accessibility element when
+the result isn't clear from the accessibility label.
+
+To use, set the ```accessibilityHint``` property to a custom string:
+
+```tsx
+<Button content="Back" accessibilityLabel="Go Back" accessibilityHint="Navigates to the previous screen" />
+```
+
+In the above example, Narrator will read the hint after the label.
+
+#### acceptsKeyboardFocus
+
+When ```true```, this enables the component to accept keyboard focus. 
+
+#### accessibilityLevel, accessibiiltyPositionInSet, accessibilitySetSize
+
+These properties are used together to define a set of items.
+- ```accessibilityLevel``` defines the level of the set
+- ```accessibilityPositionInSet``` defines the position of the item in a set
+- ```accessibilitySetSize``` defines the number of items in a set
+
+```tsx
+<View>
+  <Button content="One" accessibilityLevel={1} accessibiltyPositionInSet={1} accessibilitySetSize={2}/>
+  <Button content="Two" accessibilityLevel={1} accessibiltyPositionInSet={2} accessibilitySetSize={2}/>
+  <View>
+    <Button content="Three" accessibilityLevel={2} accessibiltyPositionInSet={1} accessibilitySetSize={3} />
+    <Button content="Four" accessibilityLevel={2} accessibiltyPositionInSet={2} accessibilitySetSize={3} />
+    <Button content="Five" accessibilityLevel={2} accessibiltyPositionInSet={3} accessibilitySetSize={3} />
+  </View>
+</View>
+```
+
+In the above example, Narrator will read the first button as "One, 1 of 2, level 1" and the second button as "Two, 2 of 
+2, level 1" and so on.
+
+### Roles
+```accessibilityRole``` communicates the purpose of the component to the user.
+
+### States
+
+```accessibilityStates``` describes the current state of a component to the user.
```

Comment:
Isn't React changing to `accessibilityState` prop?

Reply:
Yes, but we currently only have support for accessibilityStates at the moment. When we add support for accessibilityState we should update the docs.